vowpalwabbit/automl.cc

```diff
@@ -202,6 +203,34 @@ void interaction_config_manager::pre_process(const multi_ex& ecs)
   }
 }
 
+// Helper function to insert new configs from oracle into map of configs as well as index_queue.
+// Handles creating new config with exclusions or overwriting stale configs to avoid reallocation.
+void interaction_config_manager::insert_config(
```

Comment:
can this be `std::move`'d into here and then moved into the `.exclusions` field?

Reply:
Yes, changed

Reply:
which formatter? The reason is that it's a needless copy since the `new_exclusions` object is being created outside of this function with the sole purpose of being stored in the configs right?

Reply:
oh ok I see what you mean from your previous commit, in order to actually move the signature of the function needs to change too to:

`
void interaction_config_manager::insert_config(std::map<namespace_index, std::set<namespace_index>>&& new_exclusions)
`

Reply:
Right forgot the rvalue in the signature. Just added it back with some other moves